src/components/shared/customRoute/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { Redirect, Route } from 'react-router-dom';
+
```

Comment:
Remove the extra blank line

Reply:
I added to add a line between the _node_module_ imports and _local_ imports. What do you think about that?

Reply:
We didn't use to do that. but it seems a good idea. keep it please.